e2e-test/jest.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  preset: 'jest-puppeteer',
+  // jest-puppeteer manages the testEnvironment variable, so be sure not to specify it
+
+  setupFilesAfterEnv: [
+    // Normally, expect-puppeteer is specified by the jest-puppeteer preset
+    'expect-puppeteer',
+    'jest-puppeteer-vuetify',
+    './jest.setup.js',
+  ],
+
+  transformIgnorePatterns: [
+    // Apply Babel transformation to jest-puppeteer-vuetify
+    'node_modules/(?!jest-puppeteer-vuetify/)',
+  ],
```

Comment:
This is an artifact from when `jest-puppeteer-vuetify` was in a subdirectory, I think it can be safely removed now.